ios/sdk/src/JitsiMeetView.m

```diff
@@ -211,6 +216,13 @@ + (BOOL)application:(UIApplication *)application
                                annotation:annotation];
 }
 
++ (BOOL)application:(UIApplication *)app openURL:(NSURL *)url
```

Comment:
Adding this delegate method will break deep-linking because it's the new (non-deprecated way) of handling links, the method just above this one needs to be embedded here, and removed.